tensorflow/core/kernels/bincount_op_gpu.cu.cc

```diff
@@ -50,9 +50,8 @@ struct BincountFunctor<GPUDevice, Tidx, T, false> {
       return Status::OK();
     }
     if (tensorflow::OpDeterminismRequired()) {
-      // TODO(reedwm): Is this really nondeterministic? There is no
-      // documentation in DeviceHistogram::HistogramEven on whether it is
-      // deterministic or not.
+      // TODO(reedwm): Is this really nondeterministic?
+      // https://github.com/NVIDIA/cub/issues/471#issuecomment-1194682443
```

Comment:
@reedwm What do you think about this?

Reply:
I said above that this means Bincount is likely deterministic. But it turns out that is wrong. The comment in that links is:

> assuming that your histogram output is an integral type

But the histogram output can be floating-point, if the `weights` passed to `tf.math.bincount` is floating-point.

Anyway, I would still mention `DeviceHistogram::HistogramEven` for context. Maybe say something like 

    Is there really nondeterministic? DeviceHistogram::HistogramEven is called, and it is unclear if it is deterministic on floating-point inputs. See https://github.com/NVIDIA/cub/issues/471#issuecomment-1194682443.